podcasts/DiscoverViewController.swift

```diff
@@ -95,7 +95,13 @@ class DiscoverViewController: PCViewController {
         if mainScrollView.contentOffset.y.rounded(.down) > defaultOffset.rounded(.down) {
             mainScrollView.setContentOffset(CGPoint(x: 0, y: defaultOffset), animated: true)
         } else {
-            searchController.searchTextField.becomeFirstResponder()
+            // When double-tapping on tab bar, dismiss the search if already active
+            // else give focus to the search field
+            if searchController.cancelButtonShowing {
+                searchController.cancelTapped(self)
```

Comment:
- Feels a bit hacky to simulate a button tap, but not sure if it is work duplicating the code in that method or refactoring it.
- Also, should I pass `self` or `nil` or not important? More of a style question I guess.

Reply:
I think it's fine to pass `self`.

While it seems hack I think it's a ok trade-off for a gesture I assume not many people use that much. I learned of that quite recently by the way.